### ISSUE TITLE
Ubuntu 20.04 scripts + python3

### ISSUE THREAD

**Issue by kkiesling** [pyne/install_scripts#49]
We need to add a Ubuntu 20.04 script (bash and dockerfile). The default python on 20.04 is python 3 so scripts should be updated accordingly.

For Ubuntu 18.04, the default python is still python 2. We should update the dockerfile to change to python 3 but give an option in the bash script for either python 2 or 3.

We should also probably just remove 16.04 from our install scripts since that is two LTS versions removed at this point.

**Comment by kkiesling** [comment]
scratch the comment about option python 2 or 3 for 18.04- I was looking at an outdated file. We already do this. But we should probably still just switch the 18.04 dockerfiles to default python 3.

**Comment by gonuke** [comment]
Have you tried just changing the base image for the 18.04 dockerfile to use 20.04?  I can do that if you're not already on it...

**Comment by kkiesling** [comment]
For sake of record: yes, I tried changing the base on the 18.04 image and using that. 20.04 does not come with a `python` executable, only `python3`. I've updated the dockerfile scripts to call `python3` and install python3 packages instead. However, there are a few issues within the pyne source itself where `python` is called which causes an error. I believe this happens in `nuc_data_make` command. PyNE can build though, just no data. I can switch to trying to fetch prebuilt instead and see if that works. But we'll need to update PyNE to use the correct executable.
